packages/ui/src/views/chatmessage/ChatMessage.css

```diff
@@ -75,6 +75,10 @@
 }
 
 .markdownanswer a {
+    display: block;
+    margin-right: 2.5rem;
+    word-wrap: break-word;
+    overflow-wrap: break-word;
```

Comment:
dont think u need `overflow-wrap`.

somehow its still showing a little space there: 
<img width="339" alt="image" src="https://github.com/FlowiseAI/Flowise/assets/26460777/32d7bae9-ff54-4624-a715-5a8631f50677">

Reply:
Sure working on it

Reply:
@HenryHengZJ 
While prompting if i get Request failed. I also need to move the horizontal scroll bar to see the full error.

https://github.com/FlowiseAI/Flowise/assets/124426279/ce4d5b4a-7e3f-430b-a643-dbaf7ce278c3

Reply:
thanks for finding that out too! yea you can disable horizontal scrolling